marklogic-data-hub/src/test/java/com/marklogic/hub/step/impl/IngestWithSourceNameAndSourceTypeTest.java

```diff
@@ -53,8 +53,8 @@ void runDefaultIngestionWithSourceNameAndSourceType(){
         assertEquals(JobStatus.FINISHED.toString(), response.getJobStatus());
         JsonNode sourceDoc = getStagingDoc("customer1.json");
         assertNotNull(sourceDoc);
-        JsonNode sourcesNode = sourceDoc.get("envelope").get("headers").get("sources").get(0);
-        assertEquals("sample-source", sourcesNode.get("name").asText());
+        JsonNode sourcesNode = sourceDoc.get("envelope").get("headers").get("sources").get(1);
```

Comment:
Can you add another assertion on "sources[0]", verifying that it's the step name?